tests/apache/hive/hooks/test_hive.py

```diff
@@ -14,102 +14,98 @@
 TEST_METASTORE_CONN_ID = "test_conn_id"
 
 
-@mock.patch("astronomer.providers.apache.hive.hooks.hive.HiveCliHookAsync.get_connection")
-@mock.patch("airflow.configuration.AirflowConfigParser.get")
-@mock.patch("impala.hiveserver2.connect")
-def test_get_hive_client_with_conf(mock_get_connect, mock_get_conf, mock_get_connection):
-    """Checks the connection to hive client"""
-    mock_get_connect.return_value = mock.AsyncMock(HiveServer2Connection)
-    mock_get_conf.return_value = "kerberos"
-    mock_get_connection.return_value = models.Connection(
-        conn_id="metastore_default",
-        conn_type="metastore",
-        port=10000,
-        host="localhost",
-    )
-    hook = HiveCliHookAsync(TEST_METASTORE_CONN_ID)
-    result = hook.get_hive_client()
-    assert isinstance(result, HiveServer2Connection)
+class TestHiveCliHookAsync:
+    @mock.patch("astronomer.providers.apache.hive.hooks.hive.HiveCliHookAsync.get_connection")
+    @mock.patch("airflow.configuration.AirflowConfigParser.get")
+    @mock.patch("impala.hiveserver2.connect")
+    def test_get_hive_client_with_conf(self, mock_get_connect, mock_get_conf, mock_get_connection):
+        """Checks the connection to hive client"""
+        mock_get_connect.return_value = mock.AsyncMock(HiveServer2Connection)
+        mock_get_conf.return_value = "kerberos"
+        mock_get_connection.return_value = models.Connection(
+            conn_id="metastore_default",
+            conn_type="metastore",
+            port=10000,
+            host="localhost",
```

Comment:
All these values can be variables. since it's repeated in multiple places. eg `TEST_METASTORE_CONN_ID = "test_conn_id"`